lldb/unittests/DAP/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
 add_lldb_unittest(DAPTests
+  DAPTest.cpp
+  Handler/DisconnectRequestHandlerTest.cpp
```

Comment:
I somewhat regret the naming of the request handlers (*). Learning from our mistakes, maybe here we should  just call it `Handler/DisconnectTest.cpp` to reduce the verbosity?

(*) In hindsight, I wish I called them `RequestHandlerDisconnect` instead  of `DisconnectRequestHandler`, though that doesn't apply here as all the unit test files end with `Test` and I  think consistency is more important.

Reply:
I renamed the file to `Handler/DisconnectTest.cpp` but should the test be `DisconnectTest` or should it be `DisconnectRequestHandlerTest` since thats the actual class I'm testing?

Reply:
I think `Handler/DisconnectTest` is sufficiently clear.